pages/common/stty.md

```diff
@@ -0,0 +1,23 @@
+# stty
+
+> Set options for a terminal device interface.
+
+- Display all settings for the current terminal:
+
+`stty -a`
+
+- Set the number of rows:
+
+`stty rows {{rows}}`
+
+- Set the number of columns:
+
+`stty cols {{cols}}`
+
+- Get the only actual transfer speed of a device:
```

Comment:
Ah, this is much clearer - thanks! Perhaps it would read better as `Get the actual`.....?